anasymod/emu/vivado_emu.py

```diff
@@ -89,6 +89,13 @@ def build(self):
             # Setup Debug Hub
             constrs.use_templ(TemplDbgHub(target=self.target))
 
+            # Add false paths for Zynq control signals.  This is necessary in some cases
+            # to provide timing violations, since the ARM core is running on a different
+            # clock that the emulator circuitry.  This is a real problem, but is handled
+            # in firmware with handshaking and very short delays.
+            if self.target.cfg.fpga_sim_ctrl == FPGASimCtrl.UART_ZYNQ:
+                constrs.writeln('set_false_path -through [get_pins sim_ctrl_gen_i/zynq_gpio_i/*]')
```

Comment:
For now, we rely on firmware to manage synchronization between the PS and PL.  The PS firmware sets up some data, waits a short amount of time, then asserts a valid signal to write the data.  Might be good to revisit this at some point in the future to make sure that glitches cannot get through.  But this problem is not unique to ZCU106; I just noticed the timing violations (which were slowing down the FPGA build) and realized that we hadn't set false paths on the PS signals going to/from the PL.